packages/story-editor/src/components/footer/carousel/karma/carouselNavigation.karma.js

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { createSolid } from '@web-stories-wp/patterns';
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../../karma';
+import { useStory } from '../../../../app/story';
+import { useInsertElement } from '../../../canvas';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '../../../../app/font/defaultFonts';
+
+describe('Carousel Navigation', () => {
+  let fixture;
+  let element1;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    fixture.setPages([
+      { id: 'page1', backgroundColor: createSolid(255, 255, 255) },
+      { id: 'page2', backgroundColor: createSolid(255, 0, 0) },
+      { id: 'page3', backgroundColor: createSolid(0, 255, 0) },
+      { id: 'page4', backgroundColor: createSolid(0, 0, 255) },
+    ]);
+    await fixture.render();
+
+    const insertElement = await fixture.renderHook(() => useInsertElement());
```

Comment:
Could we do this by actually clicking on a text element instead? If I'm not mistaken we even had an issue somewhere to replace using hooks for element insertion by using clicking -- to make it more close to real usage.

Reply:
Fixed in acf6fc10be.